src/exchange/providers/ollama.py

```diff
@@ -16,24 +16,26 @@
 from exchange.tool import Tool
 
 OLLAMA_HOST = "http://localhost:11434/"
+OLLAMA_MODEL = "llama3.1:8b-instruct-q8_0"
 
 #
-# NOTE: this is experimental, best used with 70B model or larger if you can. 
-# Example profile config to try:
+# NOTE: this is experimental, best used with 70B model or larger if you can.
 class OllamaProvider(Provider):
     """Provides chat completions for models hosted by Ollama"""
 
-    """
-
-        ollama:
-          provider: ollama
-          processor: llama3.1
-          accelerator: llama3.1
-          moderator: passive
-          toolkits:
-          - name: developer
-            requires: {}
-    """    
+    __doc__ += f"""
+
+Here's an example profile configuration to try:
```

Comment:
added to the docstring, but happy to revert if we wanted to provide an example just in source for those glancing.

Reply:
doesn't hurt